src/main/java/com/bmc/truesight/saas/jira/integration/adapter/JiraEntryEventAdapter.java

```diff
@@ -136,6 +155,84 @@ public JIRAEventResponse eventList(JsonNode responseIssuesNode, Template templat
         }
         response.setValidEventList(tsiValidEventList);
         response.setInvalidEventList(invalidEventList);
+        response.setInvalidEventIdsList(invalidEvents);
         return response;
     }
+
+    private String getMultipleValues(FieldItem fieldItem, JsonNode jsonNode) {
+        StringBuilder value = new StringBuilder();
+        int count = 0;
+        if (jsonNode != null && !jsonNode.isNull()) {
+            for (JsonNode node : jsonNode) {
+                try {
+                    if (count == 0) {
+                        value.append(node.get(fieldItem.getFetchKey()).asText());
+                    } else {
+                        value.append(",").append(node.get(fieldItem.getFetchKey()).asText());
+                    }
+                    count++;
+                } catch (Exception ex) {
+                }
+            }
+        }
+        return value.toString();
+    }
+
+    private String getArrayValues(JsonNode jsonNode) {
+        StringBuilder value = new StringBuilder();
+        ObjectMapper mapper = new ObjectMapper();
+        if (jsonNode != null && !jsonNode.isNull()) {
```

Comment:
! and && are very complex conditions... its always better to avoid them... in this case, you may choose to do this `if(jsonNode == null || jsonNode.isNull()){ return value.toString(); }
Then, continue with other stuff after this.